core/DataTable/Filter/ColumnDelete.php

```diff
@@ -161,4 +145,46 @@ public function filter($table)
 
         return $table;
     }
+
+    /**
+     * @param $table
+     * @return array
+     */
+    protected function removeColumnsFromTable(&$table)
+    {
+        if(!$this->isArrayAccess($table)) {
```

Comment:
without it there's a notice. it's quite weird how this filter is used on both DataTable (normal flow) and arrays (in ResponseBuilder)